Format.ts

```diff
@@ -0,0 +1,47 @@
+// ----- Types ----- //
+
+const enum Pillar {
+    News,
```

Comment:
I like the upper caseness of this - it makes sense like this (less so with strings)